File: official/docs/java/webhooks/UpdateWebhook.java
import com.easypost.EasyPost;

public class UpdateWebhook {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        Webhook webhook = Webhook.retrieve("hook_...");
        webhook.update();

        System.out.println(webhook);
    }
}

File: official/docs/csharp/webhooks/UpdateWebhook.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Xunit;
using Xunit.Sdk;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples;

public class Examples
{
    [Fact]
    public async Task UpdateWebhook()
    {
        string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

        EasyPost.ClientManager.SetCurrent(apiKey);

        Webhook webhook = await Webhook.Retrieve("hook_...");

        await webhook.Update();

        new TestOutputHelper().WriteLine(JsonConvert.SerializeObject(webhook, Formatting.Indented));
    }
}
